packages/entities/entities-routes/docs/route-list.md

```diff
@@ -166,6 +166,13 @@ HTML element you want title to be rendered as. Defaults to `h2`.
 
 Whether to show the "Expression" column. Defaults to `false`.
 
+#### `enableV2EmptyStates`
+- type: `boolean`
+- required: `false`
+- default: `false`
```

Comment:
```suggestion
- default: `false`
```

We accidentally did the docs wrong and then copy/pasted everywhere. We should display either `required: true` OR `default: someValue`